benchmark/sirun/runall.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+source ~/.nvm/nvm.sh
+# source /usr/local/nvm/nvm.sh
```

Comment:
Need to conditionally source these files. The first is for macOS, the second is for Docker.